Server/Static/statichelper.py

```diff
@@ -236,7 +239,10 @@ def getActionData(version):
 
 	# Add each action name to the list
 	for actn in all_ver_actions:
-		action_dict[actn.name] = actn.description
+		action_dict[actn.name] = {}
+		action_dict[actn.name]["Object"]      = actn
+		action_dict[actn.name]["Description"] = actn.description
+		logging.debug("DESC IS {0}".format(actn.description))
```

Comment:
Left over debugging statement that we should probably remove or update to have a better message.